python/pyhail/dataset.py

```diff
@@ -930,10 +1199,115 @@ def linreg(self, y, covariates='', root='va.linreg', minac=1, minaf=None):
             pargs.append(str(minaf))
         return self.hc.run_command(self, pargs)
 
-    def logreg(self, test, y, covariates=None, root=None):
+    def logreg(self, test, y, covariates=None, root='va.logreg'):
         """Test each variant for association using the logistic regression
         model.
 
+        **Example**
+
+        Run logistic regression with Wald test with two covariates:
+
+        >>> (hc.read('data/example.vds')
+        >>>   .annotate_samples_table('data/pheno.tsv', root='sa.pheno',
+        >>>     config=TextTableConfig(impute=True))
```

Comment:
TextTableConfig is imported into module `pyhail` and I think in the code snippets we should assume the `hc` has been created and `import * from pyhail`.  Reasonable?